src/InlineValid/InlineValid.js

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { dcnb } from "cnbuilder";
+import Icon from "react-hero-icon";
+
+export const InlineValid = ({ className, validText, icon, iconType }) => {
```

Comment:
Does it make sense to create an InlineValid component to display validation messages? Or is there an elegant way we can reuse the InlineError component to toggle between Valid and Error messages/styles 🤔 

I imagine there's a way to reuse and recycle — I'll poke around RHF documentation to see how these messages are typically absorbed by the component.

Reply:
You could combine the two, but I think it's fine to keep them separate since they serve two different, mutually exclusive purposes despite the fact that they generally occupy the same space in the DOM structure.